packages/backend/src/logging/winston.ts

```diff
@@ -22,42 +23,73 @@ const colors = {
 };
 winston.addColors(colors);
 
-const addSentryTraceId = winston.format((info) => ({
-    ...info,
-    sentryTraceId: getActiveSpan()?.spanContext().traceId,
-}));
+export type SentryInfo = {
+    sentryTraceId?: string;
+};
+
+const addSentryTraceId = winston.format(
+    (info): winston.Logform.TransformableInfo & SentryInfo => ({
+        ...info,
+        sentryTraceId: getActiveSpan()?.spanContext().traceId,
+    }),
+);
+
+export type ExecutionContextInfo = {
+    worker?: {
+        id: string | null;
+    };
+    job?: {
+        id: string;
+        queue_name: string | null;
+        task_identifier: string;
+        priority: number;
+        attempts: number;
+    };
+};
+
+const addExecutionContent = winston.format(
+    (info): winston.Logform.TransformableInfo & ExecutionContextInfo =>
+        ExecutionContext.exists()
+            ? {
+                  ...info,
+                  ...ExecutionContext.get<ExecutionContextInfo>(),
+              }
+            : info,
+);
+
+const printMessage = (
+    info: winston.Logform.TransformableInfo & ExecutionContextInfo & SentryInfo,
+): string => {
+    const sentryTraceId = info.sentryTraceId
+        ? ` Sentry:${info.sentryTraceId}`
+        : '';
+
+    const workerId = info.worker?.id ? ` Worker:${info.worker.id}` : '';
+    const jobId = info.job?.id ? ` Job:${info.job.id}` : '';
+
+    return `${info.timestamp}${sentryTraceId}${workerId}${jobId} [Lightdash] ${
```

Comment:
I don't find the sentry and the worker info very useful, plus pollutes the message. Should we remove them those from the message?